### ISSUE TITLE
ServiceUnavailable from host runtime while deploying function

### ISSUE THREAD

**Issue by abhishekmishragithub** [Microsoft/vscode-azurefunctions#943]
Repro steps:
Try to deploy function from VS Code using Azure function extension 
![error_vscode_python_deploying](https://user-images.githubusercontent.com/38150419/52027586-ef4df680-2531-11e9-9ccc-d1ad33d878a3.JPG)
- Deploy icon 
Action: azureFunctions.deploy
Error type: BadRequest
Error Message: Encountered an error (ServiceUnavailable) from host runtime.

Version: 0.13.1
OS: win32

**Comment by EricJizbaMSFT** [comment]
Closing as dupe of https://github.com/Microsoft/vscode-azurefunctions/issues/804. Linux Consumption plans sometimes have outages or are slow. We added logic to retry a few times if we get a ServiceUnavailable which should mitigate this issue and be included in the next release.

I also see a problem that `func pack` doesn't have any output. See this comment for more info on that: https://github.com/Microsoft/vscode-azurefunctions/issues/892#issuecomment-459054366
